langchain-core/src/runnables/tests/runnable.test.ts

```diff
@@ -81,15 +81,15 @@ test("Create a runnable sequence with a runnable map", async () => {
     ),
   ]);
   const llm = new FakeChatModel({});
-  const inputs = {
+  const inputs = RunnableMap.from({
     question: (input: string) => input,
     documents: RunnableSequence.from([
       new FakeRetriever(),
       (docs: Document[]) => JSON.stringify(docs),
     ]),
     extraField: new FakeLLM({}),
-  };
-  const runnable = new RunnableMap({ steps: inputs })
```

Comment:
The one unintended consequence of this change is that the compiler has more trouble inferring the RunInput type when the steps are passed in as a variable, rather than passed in directly, which makes the above change necessary. I tried to fix this with no success.